internal/consensus/reactor.go

```diff
@@ -1400,10 +1399,6 @@ func (r *Reactor) processVoteSetBitsCh(ctx context.Context) {
 		envelope := iter.Envelope()
 
 		if err := r.handleMessage(ctx, r.voteSetBitsCh.ID, envelope); err != nil {
-			if errors.Is(err, context.Canceled) || errors.Is(err, context.DeadlineExceeded) {
```

Comment:
the other channels processed by handle message can return context errors, so we should leave htis.

Reply:
If we check other channel msg handle logic like `processVoteCh`,  they are the same logic. Original I was thinking every handleMessage should check the context status immediately(the first commit in my PR), but actually the following `SendError` will check the context status, therefore I think this line can be removed.

Reply:
So the only difference just should we print the error log when the context error happens.

Reply:
Ah, I am wrong. Roll back the changes.